.coveragerc

```diff
@@ -15,7 +15,10 @@ skip_covered = true
 skip_empty = true
 show_missing = true
 exclude_also =
-  ^\s*@pytest\.mark\.xfail
```

Comment:
Could you keep this? It's not related to typing and also, stable coverage numbers are more important.